tea shop/index.html

```diff
@@ -10,13 +10,13 @@
 </head>
     <header>
     <h2>A quaint little Tea Shop</h2>
-    <nav href="../">Home </nav>
     <nav href="products">Tea </nav>
-</header>
+    </header>
 <body>
 <h3>Teas</h3>
-<ul id="teas"></ul>
+
 <img src="../tea shop/assets/Emperor-Shen-Nong.jpg"       class="tea-master-image"/> 
```

Comment:
I would name your folders with kebab case. so instead of tea shop it would be tea-shop